chemprop/v2/tests/unit/featurizers/test_cgr.py

```diff
@@ -145,9 +145,9 @@ def test_get_function_lowercase_mode(self, mode_name):
         """
         Test that the get function returns the correct RxnMode when given a lowercase mode.
         """
-        assert RxnMode.get(mode_name.lower()) == getattr(RxnMode, mode_name)
-        assert RxnMode.get(mode_name.lower()).name == mode_name
-        assert RxnMode.get(mode_name.lower()).value == mode_name.lower()
+        assert RxnMode.get(mode_name) == getattr(RxnMode, mode_name)
+        assert RxnMode.get(mode_name).name == mode_name
+        assert RxnMode.get(mode_name).value == mode_name.lower()
```

Comment:
In the description for this test, it mentions "when given a lowercase mode".  Here it looks like you removed the part that makes it lowercase. Now this function `test_get_function_lowercase_mode` is identical to the function above it `test_get_function`, which I assume you didn't mean to do.